paths/answers/comments.yaml

```diff
@@ -1,23 +1,20 @@
 x-swagger-router-controller: mock
-post:
+get:
   description: |
-    Create a Comment on Answer
+    Request all Comments of the answer
   parameters:
-    - $ref: '../../parameters/X-User-Token.yaml'
     - name: uuid
       in: path
-      description: UUID of the Answer
+      description: Id of the answer
```

Comment:
is it the id?

Reply:
Missed this one, don't even know why I made that change